client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import { isObject } from 'lodash';
+import React, { PureComponent, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PostTime from 'reader/post-time';
+
+class EditorRevisionsListItem extends PureComponent {
+	selectRevision = event => {
+		const revisionId = parseInt( event.currentTarget.dataset.revisionId, 10 );
```

Comment:
nice catch, leftover from when it wasn't a component on its own ;) fixed in the latest update